src/kili/services/plugins/upload.py

```diff
@@ -62,6 +64,26 @@ def check_file_is_txt(path: Path, verbose: bool = True) -> bool:
     return check_file_mime_type(path, mime_extensions_for_txt_files, verbose)
 
 
+def check_file_contain_handler(name: str, path: str):
+    """
+    Return true if the file contain PluginHandler Class
+    """
+    spec = importlib.util.spec_from_file_location(name, path)
+    if not spec:
+        return False
+    plugin = importlib.util.module_from_spec(spec)
+    sys.modules[name] = plugin
+    loader = spec.loader
+    if not loader:
+        return False
+    loader.exec_module(plugin)
```

Comment:
try leaving some empty lines to make your code more readable

Reply:
done !